http-client/src/main/java/io/airlift/http/client/jetty/JettyHttpClient.java

```diff
@@ -1291,4 +1328,30 @@ public RequestDistribution(HttpClient httpClient, Processor processor)
             });
         }
     }
+
+    private static class SecureHttpClient
+            extends HttpClient
```

Comment:
This is not a good idea. It's not clear that HttpClient was designed to be extended like this.